drivers/filesystems/fastfat_new/allocsup.c

```diff
@@ -4678,7 +4678,7 @@ Return Value:
 {
     UCHAR Log = 0;
 
-#if FASTFATDBG
+#ifdef FASTFATDBG  // __REACTOS__: Use '#ifdef'.
```

Comment:
`#if <undefined preprocessor symbol>` is correct and well-defined. Stop introducing ROS-diffs for it!

Reply:
I am not sure what your point is:
if one uses `#define symbol 0`, then `#if` behaves differently/wrongly than `#ifdef`, does it not?

Reply:
Correct. And if `#define symbol 0` is an option, `#if` is the only correct way to check. Any symbol used with `#ifdef` must never be defined to 0, because that would be extremely confusing.
Therefore, `#ifdef` -> `#if` changes are important (because `-DDBG=0` is actually something that happens), whereas `#if` -> `#ifdef` changes are not (because `-DFASTFATDBG=0` would mean that all checks should use `#if`, not the other way around).